src/VERSION

```diff
@@ -1 +1 @@
-1.1.0
+1.2.0-dev
```

Comment:
Should this be 1.1.1? I think we have at least one fix for that version.

Reply:
The master branch should be heading towards the next minor release. We shouldn't keep it frozen any longer (it's been a month now).

If we need patch releases, they would be backported to the `release/1.1` branch.
What fix do you have in mind?

Reply:
Ah, I guess #10960?

Reply:
Yes, that one. But I can't remember how branches and releases work, so better to ignore me 😅